### PR TITLE
New package: r-hh

### PR DESCRIPTION
**Pull request by glennpj**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-hh/package.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RHh(RPackage):
+    """Support software for Statistical Analysis and Data Display (Second
+    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/r-hh/package.py]:
```suggestion
    """Support software for Statistical Analysis and Data Display.
    
    Support software for Statistical Analysis and Data Display (Second
    Edition, Springer, ISBN 978-1-4939-2121-8, 2015) and (First Edition,
```

**Review by alalazo — DISMISSED** [review]
LGTM, just a request to add a short first paragraph to the docstring (it will be used where a short description of the package is needed by commands or modules).

File: var/spack/repos/builtin/packages/r-hh/package.py (hunk 1 above)
Comment (glennpj) [reply var/spack/repos/builtin/packages/r-hh/package.py]: I did not realize that was parsed that way. I see that first line gets uses by `module whatis`. Which spack commands use the short description?

I will use the one liner from the project page.

Thanks.

**Review by adamjstewart — APPROVED** [review]

File: var/spack/repos/builtin/packages/r-hh/package.py (hunk 1 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/r-hh/package.py]: I believe the module files Spack generates use the short description, while `spack info` uses the full description.

**Pull request merged**